base/ubi10/Dockerfile

```diff
@@ -0,0 +1,200 @@
+    cd "${TEMP_DIR}"; \
+    RG_VERSION="13.0.0"; \
+    if [ "$TARGETARCH" = "arm64" ]; then \
+        RG_ARCH="arm-unknown-linux-gnueabihf"; \
+    else \
+        RG_ARCH="x86_64-unknown-linux-musl"; \
+    fi; \
+    RG_TGZ="ripgrep-${RG_VERSION}-${RG_ARCH}.tar.gz"; \
+    RG_TGZ_URL="https://github.com/BurntSushi/ripgrep/releases/download/${RG_VERSION}/${RG_TGZ}"; \
+    curl -sSLO "${RG_TGZ_URL}"; \
+    tar -zxv --no-same-owner -f "${RG_TGZ}"; \
+    mv "ripgrep-${RG_VERSION}-${RG_ARCH}"/rg /usr/local/bin/; \
+    mv "ripgrep-${RG_VERSION}-${RG_ARCH}"/doc/rg.1 /usr/local/share/man/man1; \
+    cd -; \
+    rm -rf "${TEMP_DIR}"
+
+# Download and install bat depending on the architecture.
+# See release page for details https://github.com/sharkdp/bat/releases/tag/v0.18.3
+RUN \
+    TEMP_DIR="$(mktemp -d)"; \
+    cd "${TEMP_DIR}"; \
+    BAT_VERSION="0.18.3"; \
+    if [ "$TARGETARCH" = "arm64" ]; then \
+        BAT_ARCH="aarch64-unknown-linux-gnu"; \
+    else \
+        BAT_ARCH="x86_64-unknown-linux-musl"; \
+    fi; \
+    BAT_TGZ="bat-v${BAT_VERSION}-${BAT_ARCH}.tar.gz"; \
+    BAT_TGZ_URL="https://github.com/sharkdp/bat/releases/download/v${BAT_VERSION}/${BAT_TGZ}"; \
+    curl -sSLO "${BAT_TGZ_URL}"; \
+    tar -zxv --no-same-owner -f "${BAT_TGZ}"; \
+    mv "bat-v${BAT_VERSION}-${BAT_ARCH}"/bat /usr/local/bin/; \
+    mv "bat-v${BAT_VERSION}-${BAT_ARCH}"/bat.1 /usr/local/share/man/man1; \
+    cd -; \
+    rm -rf "${TEMP_DIR}"
+
+# Download and install fd depending on the architecture.
+# See release page for details https://github.com/sharkdp/fd/releases/tag/v8.7.0
+RUN \
+    TEMP_DIR="$(mktemp -d)" && \
+    cd "${TEMP_DIR}" && \
+    FD_VERSION="8.7.0" && \
+    if [ "$TARGETARCH" = "arm64" ]; then \
+        FD_ARCH="aarch64-unknown-linux-gnu"; \
+    else \
+        FD_ARCH="x86_64-unknown-linux-musl"; \
+    fi && \
+    FD_TGZ="fd-v${FD_VERSION}-${FD_ARCH}.tar.gz" && \
+    FD_TGZ_URL="https://github.com/sharkdp/fd/releases/download/v${FD_VERSION}/${FD_TGZ}" && \
+    curl -sSLO "${FD_TGZ_URL}" && \
+    tar -xv --no-same-owner -f "${FD_TGZ}" && \
+    mv "fd-v${FD_VERSION}-${FD_ARCH}"/fd /usr/local/bin && \
+    mv "fd-v${FD_VERSION}-${FD_ARCH}"/fd.1 /usr/local/share/man/man1 && \
+    cd - && \
+    rm -rf "${TEMP_DIR}"
+
+# Define user directory for binaries
+ENV PATH="/home/user/.local/bin:$PATH"
+
+# Set up environment variables to note that this is
+# not starting with usernamespace and default to
+# isolate the filesystem with chroot.
+ENV _BUILDAH_STARTED_IN_USERNS="" BUILDAH_ISOLATION=chroot
+
+# Tweaks to make rootless buildah work
+RUN touch /etc/subgid /etc/subuid  && \
+    chmod g=u /etc/subgid /etc/subuid /etc/passwd  && \
+    echo user:10000:65536 > /etc/subuid  && \
+    echo user:10000:65536 > /etc/subgid
+
+# Adjust storage.conf to enable Fuse storage.
+RUN sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' /usr/share/containers/storage.conf
```

Comment:
In the `base/ubi9` image, I noticed that the `storage.conf` file path `/etc/containers/storage.conf`

Is this location supposed to be different for  UBI9 and UBI10?